src/main/java/org/sagebionetworks/web/client/EntityTypeUtils.java

```diff
@@ -127,6 +134,8 @@ public static IconType getIconTypeForEntityClassName(String className) {
 		} else if (Dataset.class.getName().equals(className)) {
 			// Dataset
 			icon = IconType.TABLE;
+		} else if (DatasetCollection.class.getName().equals(className)) {
```

Comment:
Surprised that this still exists.  Could you open up an SWC ticket to remove references to getIconTypeForEntityClassName().  I think Provenance is the only place that still uses this, and it should just use EntityIcon from SRC

Reply:
It is indeed only used by Provenance, I checked before adding this. Will do